src/foremast/elb/create_elb.py

```diff
@@ -165,14 +165,18 @@ def add_stickiness(self):
             if listener.get("stickiness"):
                 sticky_type = listener['stickiness']['type'].lower()
                 externalport = int(listener['loadbalancer'].split(":")[-1])
-                policyname = "{0}-{1}cookie-{2}".format(self.app, sticky_type, externalport)
                 if sticky_type == 'app':
+                    cookiename = listener['stickiness']['cookie_name']
+                    policyname = "{0}-{1}cookie-{2}-{3}".format(self.app, sticky_type, externalport, cookiename)
```

Comment:
ohhh, I see what you mean. I hadnt thought about defining the string and then applying format later but that makes perfect sense. I'll update.